source/grid/grid_tools.cc

```diff
@@ -364,6 +364,31 @@ namespace GridTools
   }
 
 
+
+  template <int dim, int spacedim>
+  BoundingBox<spacedim>
+  compute_bounding_box(const Triangulation<dim, spacedim> &tria)
```

Comment:
what should be an expected behaviour for `p::d::Tria`? Maybe throw an error if `ExcNotImplemented()` if it's provided?

Reply:
Good point! Should I modify the function that I'm adding to deal with distributed triangulations, or the previous function that accepts an arbitrary predicate? You could ask the same question about both.

Reply:
Maybe this one only as to me it suggest the box and f the whole thing, not only of locally owned DoFs. Or maybe just mention it in the docu as a warning.

Reply:
Ps. The function you re-use, in my interpretation, does/should not promise to do the union of the results over MPI communicator